pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go

```diff
@@ -733,3 +734,13 @@ func NewPostgresCluster() *PostgresCluster {
 	cluster.SetGroupVersionKind(GroupVersion.WithKind("PostgresCluster"))
 	return cluster
 }
+
+const LabelVersion = "pgv2.percona.com/version"
```

Comment:
Can you reuse this: https://github.com/percona/percona-postgresql-operator/blob/34f1bd17199d2fa7fe73643230b8f58c621a88d4/internal/naming/labels.go#L28

Reply:
unfortunately, no. it causes cyclical import